single-product.css

```diff
@@ -0,0 +1,78 @@
+.single-product-page-container .nav-bar {
+    margin-bottom: 2rem;
+}
+
+.single-product-page-container .sidebar-products-container {
+    display: flex;
+}
+
+.single-product-page-container .sidebar-products-container .sidebar-container {
+    min-width: 25rem;
+    padding: 0 3rem;
+    border-right:1px solid var(--button-floatHover-color);
+}
+
+.single-product-page-container .sidebar-products-container .product-card-container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin-top: 1rem;
+    flex-grow: 1;
+    padding: 0 2rem 0 3rem;
+    height: auto;
+    gap:2rem;
+}
```

Comment:
Question- The specificity of these is way too much any reason why?

Reply:
So that none of the stylings apply in the above element from Component library, and is it  a bad practise though?